src/img/icons.json

```diff
@@ -19,6 +19,7 @@
     "circle-back",
     "circle-check",
     "circle-close",
+    "circle-close-error",
```

Comment:
So this is just like `circle-close`, only it's red ? Couldn't that be achieved from css with `fill` ?

Reply:
The lines of this icon are bolder, or wider than `circle-close`